server/odc-service/src/main/java/com/oceanbase/odc/service/task/schedule/daemon/CheckRunningJob.java

```diff
@@ -54,51 +62,86 @@ public void execute(JobExecutionContext context) throws JobExecutionException {
         TaskFrameworkProperties taskFrameworkProperties = getConfiguration().getTaskFrameworkProperties();
         int size = taskFrameworkProperties.getSingleFetchCheckHeartTimeoutJobRows();
         int heartTimeoutPeriod = taskFrameworkProperties.getJobHeartTimeoutSeconds();
+        handleGeneralHeartTimeoutJobs(size, heartTimeoutPeriod);
+        if (taskFrameworkProperties.getRunMode().isProcess()) {
+            handleNotLocalAndProcessIsNotExitsJobs(size, heartTimeoutPeriod);
+        }
+    }
+
+    private void handleGeneralHeartTimeoutJobs(int size, int heartTimeoutPeriod) {
```

Comment:
what's the unit of this param? `heartTimeoutPeriod`

Reply:
okey, heartTimeoutPeriod  -> heartTimeoutSeconds